source/_patterns/molecules/navigation/breadcrumbs/breadcrumbs.yml

```diff
@@ -0,0 +1,5 @@
+breadcrumbs:
+  - url: "#"
```

Comment:
`url_attributes` would be in here.  You don't need to add them in the pattern lab data (what you have is fine), just making a note that each link would have its own.